### PR TITLE
Fix const reading of State parameters

### PR DESCRIPTION
**Pull request by domire8**
Small fix as discussed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Release Versions:
 - Catch core and component exceptions (#111)
 - Modulo components documentation (#112)
 - Versioning (#113)
+-  Fix const reading of State parameters (#114)
```
Comment (domire8) [inline CHANGELOG.md:78]: sorry for this whitespace error, but we can fix it with the release

File: source/modulo_core/test/python/translators/test_parameters.py [hunk 2]
```diff
@@ -94,59 +94,11 @@ def test_state_parameter_read_write(state_parameters):
...
 
-        new_ros_param = write_parameter(param)
-        assert ros_param.name == new_ros_param.name
-        assert ros_param.type_ == new_ros_param.type_
-        assert_np_array_equal(
-            clproto.decode(clproto.from_json(new_ros_param.get_parameter_value().string_value)).data(),
-            value_types[0].data())
-
-
-def test_vector_parameter_read_write():
```
Comment (eeberhard) [inline source/modulo_core/test/python/translators/test_parameters.py:108]: please bring back these tests, I worked hard on them 🙏

**Review by eeberhard — APPROVED** [review]
👍

File: source/modulo_core/test/python/translators/test_parameters.py (hunk 2 above)
Comment (domire8) [reply source/modulo_core/test/python/translators/test_parameters.py]: except that I wrote them, even though you inspired me with your C++ tests :wink:

Comment (eeberhard) [reply source/modulo_core/test/python/translators/test_parameters.py]: Ah yes, you are the Python ying to my C++ yang ☯️

**Pull request merged**